data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Between the mundane and cultural, you find a few interesting facts. There is some lingering frustration about the Quarg refusing to help fend off Korath raids. This is partly due to the Hai having to fight for themselves, but mostly out of a belief that more Korath would have survived the war if the Quarg had intervened. Also, the Hai apparently have no interest in developing the jump drives Quarg have in abundance due to "the fate of the Korath," and warnings from the Quarg about the dangers of the jump drive manufacturing process.`
+				goto reading
+
+			label militech
+			action
+				log `A dry Hai military report doubted the assumed superiority of Hai military technology, other than their hulls. In particular, it is fearful of human weapons that do heat damage and or push a ship, warning that they can paralyze Hai vessels. They're also worried about recent human weapons development, and the lack of Hai ramscoops.`
+				set "Hai Reveal [A11] Sound the Alarm: militech book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	Predictably, the report begins with Hai hulls, which it claims are superior to human hulls by every measure. However, it warns that there are no other fields in which Hai clearly lead, and warns that the humans may soon surpass the Hai in military technology.`
+			`	Hai have a perceived advantage in several fields which, the report says, are minimal at best. Hai engines are more efficient than human atomic engines, but no faster, and humans have ion and plasma alternatives. Likewise, human atomic generators are hotter-running than those of the Hai, but humans have far more options for power. Hai shields were vastly superior, until the Syndicate's recent advancements. Hai anti-missile systems' superiority is merely a myth; human systems are as effective per ton. Of course, Hai cooling and ramscoop technology is primitive compared to humans, even with the return of the Ladybug.`
+			`	What is most concerning to the Hai is the inferiority of their weapons. They note that the ion cannon is effective against humans only because human ships rarely pack large batteries. Other Hai weapons are only marginally better than human alternatives. Meanwhile, a few plasma cannons and flame-throwers can easily shut down a Shield Beetle. Striking a beetle from 45 degrees to the front with four particle cannons will completely paralyze it until it is disabled. Recent human missiles are quite deadly, as are electron beams and plasma turrets. In all, the Hai are quite fearful of human weapons and question whether they could win an all-out war without Unfettered advancements.`
```

Comment:
~~You really want to give the player a manual on how to disable SB?~~
The particle cannon one is a bit too weird and unlikely to happen to be stated here dont you think?

Reply:
I do think the Hai have some more powerful weapons in store though, that would probably not be required to fight unfettered, but in the event of a major conflict they should perhaps mention needing to use them or smth, idk
I imagine stuff like duo-pulse, paralyzer (short range ion shock weapon inspired by them exiles) and longer range pulse too, which they would not mention to you like that but they should probably mention needing to produce them if needed. I imagine the reason why they dont is because those are expensive in terms of materials and the like (which the Unfettered dont care about as much)

Also they should mention their trackers give them an edge given humans have no real counter against those

Reply:
This is an intelligence report from the Hai to a leader of the Free Worlds (or Syndicate or Republic) and it is intended to give the humans a false sense of security. The fact that it isn't 100% complete or 100% accurate is entirely intentional.

Reply:
I can add a second report with a rebuttal if you give me an outline. Having the Hai represent themselves as massively overpowered is another reasonable approach to an intelligence report in this situation. Certainly, the weapons they have now are laughable at best.

Reply:
If they want you to underestimate them then they want war? The unfettered's idea are so deep in Hai government then?

Reply:
I really can't justify the Fettered claiming to have good military tech when their current tech is junk. Perhaps I can contrive another report to put in its place?

Reply:
I have removed this report, and I will add another in its place when I think of one.